Conceptpower+Spring/src/main/java/edu/asu/conceptpower/servlet/web/ConceptDeleteController.java

```diff
@@ -139,7 +143,7 @@ public ModelAndView confirmlDelete(@PathVariable("id") String id,
             model.setViewName("redirect:/login");
             return model;
         }
-        model.setViewName("/auth/conceptlist/concepts");
+        model.setViewName("forward:/auth/" + listName + "/concepts");
```

Comment:
if you're forwarding here, you don't need to retrieve the concepts first (line 135-139), do you?